### PR TITLE
replace vnic with nic

### PR DESCRIPTION
**Pull request by lmandavi**
It was agreed that the term "vNIC" be replaced with the term "NIC.' The CNV user guide doc was updated to reflect this change.

### CODE REVIEW

File: _topic_map.yml
Comment (ahardin-rh) [inline _topic_map.yml]: You have some extra spacing here that could be causing the build to fail

Comment (vikram-redhat) [reply _topic_map.yml]: This is most likely the cause of the build error.

File: _topic_map.yml
Comment (vikram-redhat) [inline _topic_map.yml]: If a file name for an assembly is changing, we will need to add redirects as well so folks don't get 404s. @lmandavi when you are ready, can you reach out to me about it?

File: _topic_map.yml
Comment (lmandavi) [reply _topic_map.yml]: Removed the extra spaces preceding File: nodes-nodes-managing.

**Comment by lmandavi** [comment]
The various topic modules where vNIC was referenced have been updated to reflect references to NIC. Most of these changes were within the networking topics:

https://remove-vnic--ocpdocs.netlify.com/openshift-enterprise/latest/cnv/cnv_virtual_machines/cnv_vm_networking/cnv-installing-qemu-guest-agent.html

Requesting technical review and approval.

**Comment by dankenigsberg** [comment]
> It was agreed that the term "vNIC" be replaced with the term "NIC.' The CNV user guide doc was updated to reflect this change.

"It was agreed" is not a very convincing reason. Keeping consistency is one.

@phoracek and team should review this  PR

**Comment by phoracek** [comment]
CC @AlonaKaplan

**Review by phoracek — APPROVED** [review]
LGTM. Kudos for dropping trailing spaces!

**Comment by lmandavi** [comment]
Requesting peer review.

**Comment by lmandavi** [comment]
This PR requires review and approval by QE. Requesting Alona for QE review and approval.

**Comment by yossisegev** [comment]
LGTM (CNV network QE)

**Comment by lmandavi** [comment]
Yossi Segev has completed QE review and sent approval email.

**Comment by lmandavi** [comment]
Requesting peer review.

**Comment by aburdenthehand** [comment]
LGTM - @lmandavi please squash and I will merge.
What versions does this need to be cherrypicked to?

**Comment by lmandavi** [comment]
@aburdenthehand  this is for CNV 2.3.

**Pull request merged**
